firmware/controllers/algo/launch_control.h

```diff
@@ -25,9 +26,10 @@ class LaunchControlBase {
 	bool isInsideTpsCondition() const;
 	bool isInsideSwitchCondition() const;
 	bool isInsideRPMCondition(int rpm) const;
-
 	bool isLaunchConditionMet(int rpm) const;
 
 private:
-	efitick_t launchTimer = 0;
+	efitick_t launchTimer;
```

Comment:
does this need to be initialized to 0?

Reply:
it dose not mater, we cannot use the value without it being initialized before